game/modes.py

```diff
@@ -0,0 +1,143 @@
+from .board import board_matrix as initial_board, get_options, show, update_board
+from .status import check_status
+from .player import get_current_player
+import logging
+import random
+import copy
+
+
+logger = logging.getLogger(__name__)
+
+
+def _find_winning_cell(board_matrix, available_options, sign):
```

Comment:
Is there a specific reason for using an underscore in the beginning of the function's name?
I think it should be something like:
```suggestion
def find_winning_cell(board_matrix, available_options, sign):
```